README.md

```diff
@@ -1,2 +1,40 @@
 # ember-cli-htmlbars-inline-precompile-codemod
-Codemod away from importing from htmlbars-inline-precompile to ember-cli-htmlbars.
+
+[![Build Status](https://travis-ci.com/ember-codemods/ember-cli-htmlbars-inline-precompile-codemod.svg?branch=master)](https://travis-ci.com/ember-codemods/ember-cli-htmlbars-inline-precompile-codemod)
+[![npm version](https://badge.fury.io/js/ember-cli-htmlbars-inline-precompile-codemod.svg)](https://badge.fury.io/js/ember-cli-htmlbars-inline-precompile-codemod)
+
+A collection of codemod's for ember-cli-htmlbars-inline-precompile-codemod.
+
+## Usage
+
+To run a specific codemod from this project, you would run the following:
+
+```
+npx ember-cli-htmlbars-inline-precompile-codemod <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
+
+# or
+
+yarn global add ember-cli-htmlbars-inline-precompile-codemod
+ember-cli-htmlbars-inline-precompile-codemod <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
+```
```

Comment:
Since we only really intend to ever have one codemod, I think we should tweak this to avoid requiring the transform name as an argument. I'll comment inline in the `bin/cli.js` where you'd make that change in code, but the README here will need to be tweaked as well.

The runnable snippet would look something like this:

```
npx ember-cli-htmlbars-inline-precompile-codemod path/of/files/ or/some**/*glob.js

# or

yarn global add ember-cli-htmlbars-inline-precompile-codemod
ember-cli-htmlbars-inline-precompile-codemod path/of/files/ or/some**/*glob.js
```